NYPLCardCreator/Classes/NameAndEmailViewController.swift

```diff
@@ -77,10 +77,23 @@ final class NameAndEmailViewController: FormTableViewController {
       animated: true)
   }
   
+  private func emailIsValid() -> Bool {
```

Comment:
As we discussed, email addresses can be really complicated and this regex will exclude valid email addresses. See https://davidcel.is/posts/stop-validating-email-addresses-with-regex/ for some thoughts on this topic. Personally, I would just go back to the old approach of making sure `@` is present and that's it. The user will get a chance to review their email address at the end anyway.